pycaching/trackable.py

```diff
@@ -17,7 +17,10 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
         except AttributeError:
             logging.debug("Lazy loading: %s", func.__name__)
-            self.geocaching.load_trackable(self.tid, self)
+            try:
+                self.geocaching.load_trackable_by_url(self.trackable_page, self)
+            except AttributeError:
+                self.geocaching.load_trackable(self.tid, self)
```

Comment:
I would prefer slightly modified 2nd way. Just add simple condition determining if `trackable_page` or `tid` is set and call appropriate method. If neither, raise a `pycaching.errors.LoadError`.